### ISSUE TITLE
doing "eslint --init" and selecting node does not generate that environment

### ISSUE THREAD

**Issue by wiki-me** [eslint/eslint#13967]
**Tell us about your environment**

* **ESLint Version:** v7.16.0
* **Node Version:** v14.9.0
* **npm Version:** v6.14.9

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**

default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

no configuration is made

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
npx eslint --init
```

and selected the "node" environment.

**What did you expect to happen?**

that node line will be included in the generated file " "node": true" .

**What actually happened? Please include the actual, raw output from ESLint.**

the "env" section is not correct:

```
    "env": {
        "browser": true,
        "es2021": true
    },

```

**Are you willing to submit a pull request to fix this bug?**

no.

**Comment by yeonjuan** [comment]
Hi :) @wiki-me 
Could you give us a way to reproduce it? I tried but it's not occurred
Probably, you may have hit the ENTER key without selecting the `node` on below stage.
![스크린샷 2020-12-30 오전 4 12 19](https://user-images.githubusercontent.com/41323220/103308255-4f6ab180-4a55-11eb-909c-d3fd65ff53a8.png)

**Comment by wiki-me** [comment]
I went a head and tried it in a text only shell (no gui on Kubuntu), it then works as expected (shows the Browser environment is selected with the "V sign"), but when i try it in "konsole" (under both fish shell and bash), it does not show it is selected (but pressing space and enter on the node environment generates it as expected).

**Comment by wiki-me** [comment]
To illustrate this is what it shows:

![Screenshot_20201230_194517](https://user-images.githubusercontent.com/68199012/103371436-1b84a000-4ad8-11eb-9e3f-559007ab14a9.png)

But when i press enter and finish the wizard it generates the browser as one of the environments.

**Comment by yeonjuan** [comment]
@wiki-me Sorry for the delay. it seems not an "eslint" issue.
ESLint uses [enquire](https://github.com/enquirer/enquirer) for "--init" command's prompt interaction and It may be a bug of "enquire".
